04/app.js

```diff
@@ -1,5 +1,55 @@
 console.log('DOM');
 
+
+const grand = document.querySelector('nav')
+if(grand){
+    const ul = document.createElement('ul');
+    
+    grand.appendChild(ul);
+
+    const par = document.querySelector('ul');
+
+    if(par) {
```

Comment:
Skoro parę linii kodu wyżej utworzyliśmy zmienną i ją dodaliśmy do DOM to nie trzeba jej wyszukiwać, ale od razu wykorzystać tj.
```
const grand = document.querySelector('nav')
if(grand){
    const ul = document.createElement('ul');
    
    grand.appendChild(ul);

    const par = ul; // w zasadzie wszędzie można wykorzystać nazwę ul, ale to zmieni cały kod

    if(par) {
```